packages/alfa-aria/src/feature.ts

```diff
@@ -160,7 +160,7 @@ const labels = Cache.empty<Node, Sequence<Element>>();
 const nameFromLabel = (element: Element, device: Device, state: Name.State) => {
   const root = element.root();
 
-  const elements = root.inclusiveDescendants().filter(isElement);
+  const elements = root.elementDescendants();
```

Comment:
Is it on purpose that `.inclusiveDescendants()` is gone? It looks like it means the root element won't be included anymore, but I might be overlooking something